lib/iris/unit.py

```diff
@@ -1823,9 +1823,9 @@ def convert(self, value, other, ctype=FLOAT64):
             >>> import numpy as np
             >>> c = unit.Unit('deg_c')
             >>> f = unit.Unit('deg_f')
-            >>> print(c.convert(0, f))
+            >>> print(np.round(c.convert(0, f), 8))
```

Comment:
The rounding is troublesome. I think it gives usesr the impression that somehow the rounding is actually necessary, whereas it is actually just an annoying testing issue to do with subtle behaviour changes in Python versions. Is there another option?